compiler/modules/front.syntax/tests/syntaxNode.bal

```diff
@@ -0,0 +1,465 @@
+function syntaxNodeFromImportDecl(ImportDecl decl) returns NonTerminalSyntaxNode {
+    SyntaxNode[] childNodes = [];
```

Comment:
Please write it in a more functional style without `push`. If you can't see how to do it, ask.